Sources/KituraContracts/Contracts.swift

```diff
@@ -22,7 +22,7 @@
  and server side (eg Kitura) of the request (typically a HTTP REST request).
 
  ### Usage Example: ###
- ````
+ ````swift
```

Comment:
It would be helpful to have an example for `public protocol QueryParams: Codable {}`